README.md

```diff
@@ -22,9 +22,9 @@ But starting with 1.0 everything has been moved to github and the new website.
 
 ## Motivation
 
-You might wonder, why create another Json databinding lib for Java?
-Well...most libraries, miss of important features or have a lot of features but you can hardly add new features by yourself.
-Gensons initial motivation is to solve those problems by trying to come with useful features out of the box and stay as much as possible open to extension.
+Why create another Json databinding library for Java?
+Many libraries lack either important features or the flexibility to add new ones.
+To solve such problems, Genson already contains many useful features and is being developed with future expansions in mind.
```

Comment:
Can you change this sentence to something like "Genson has been designed with this in mind, and provides out of the box many useful features but also enables users to easily customize and plug in their own extensions.". I feel it captures better the intention. The goal is to allow users to implement them self whatever is missing.

Reply:
Done. What do you think?